README.md

```diff
@@ -181,6 +181,15 @@ versioning](https://semver.org/). In other words:
 - Changes which break backwards compatibility: Major release, increment the
   first number, e.g. 2.0.0
 
+Bumping versions
+--------
+1. Starting in the  `development`  branch, update the version number in the `package.json`  on line 3
```

Comment:
I would personally not include "on line 3" just in case it changes (though I highly doubt it). What I may do instead is "update the version number (`version`) in..." or "update `version` in".

Reply:
I'd also call out 1). that we use semver to define the next version number and 2). consider linking to [this](https://github.com/crdschurch/crds-styles#versioning)